timestamp/rpc.go

```diff
@@ -0,0 +1,34 @@
+package timestamp
+
+import (
+	"bytes"
+	"time"
+
+	"github.com/centrifuge/go-substrate-rpc-client"
+	"github.com/centrifuge/go-substrate-rpc-client/scale"
+)
+
+func Now(client substrate.Client) (*time.Time, error) {
+	m, err := client.MetaData(true)
+	if err != nil {
+		return nil, err
+	}
+
+	key, _ := substrate.NewStorageKey(*m, "Timestamp", "Now", nil)
+	s := substrate.NewStateRPC(client)
+	res, err := s.Storage(key, nil)
```

Comment:
same comment as above. seems similar to AccountNonce/BlockHash - anything to generalize to avoid copy/paste?

Reply:
yup, let me try.

Reply:
we have to compromise strong types for params in order to make it more generic than this (a problem in golang in general).